esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -589,6 +601,24 @@
     lat/lon on x-axis.
 
 
+Configuration options for plot type ``benchmarking_annual_cycle``
```

Comment:
From the default options that are set below, it looks like there are indeed some configuration options for these plot types, e.g., `gridline_kwargs` for `benchmarking_annual_cycle`. It would be great to list them here or at least mention that they are the same (I guess?) as for non-benchmark version.

Reply:
Added description for all benchmarking plots.